src/hotspot/cpu/x86/x86.ad

```diff
@@ -1898,6 +1898,10 @@ const int Matcher::min_vector_size(const BasicType bt) {
   int max_size = max_vector_size(bt);
   // Min size which can be loaded into vector is 4 bytes.
   int size = (type2aelembytes(bt) == 1) ? 4 : 2;
+  // Support for calling svml double64 vectors
+  if (bt == T_DOUBLE) {
+    size = 1;
+  }
```

Comment:
@DamonFool Would T_LONG also benefit from similar change?

Reply:
> @DamonFool Would T_LONG also benefit from similar change?

Thanks @sviswa7 for your review.

Yes.
But since there is no svml support for T_LONG, I'd like to support more Double64 and Long64 vectorization after this PR.
Thanks.

Reply:
Sounds good, thanks.